server/controllers/articleController.js

```diff
@@ -1,11 +1,17 @@
 import uuid from 'uuid';
 import sequelize from 'sequelize';
 import models from '../db/models';
-import utils from '../helpers/Utilities';
 import Paginate from '../helpers/paginate';
+import Utilites from '../helpers/Utilities';
 
 const { Op } = sequelize;
 const { paginateArticles } = Paginate;
+const { errorStat, successStat } = Utilites;
+
+const parseBool = (string) => {
```

Comment:
😂🤣
Nice one @Hadeneekeh . I think this should be added to JavaScript

Reply:
Hahahahaha